autoload/fern/internal/drawer/hover_popup.vim

```diff
@@ -0,0 +1,46 @@
+function! fern#internal#drawer#hover_popup#init() abort
+  if g:fern#disable_drawer_hover_popup
+    return
+  endif
+
+  if !exists('*popup_create')
```

Comment:
Please warn users in this case for

1. Neovim is not supported (because popup_create does not exist)
2. This warning message is suppressed by `g:fern#disable_drawer_hover_popup`

Reply:
What's the best way to notify the user? I assumed just a warn-level message with `fern#logger#warn()` would be sufficient, but if there's a better way I'll change!

Reply:
`fern#logger#warn()` is OK